sdks/python/README.md

```diff
@@ -2,18 +2,105 @@
 
 The Redactive Python SDK provides a robust and intuitive interface for interacting with the Redactive platform, enabling developers to seamlessly integrate powerful data redaction and anonymization capabilities into their Python applications.
 
-## Building the Python SDK
+## Installation
 
-`cd sdks/python`
+You don't need this source code unless you want to modify the package. If you just
+want to use the package, just run:
 
-`python -m pip install --upgrade build`
+```sh
+pip install --upgrade redactive
+```
 
-`python -m build`
+Install from source with:
 
-`python install -e .`
+```sh
+python -m pip install .
+```
 
-## Building the optional reranker
+### Requirements
 
-`cd sdks/python`
+- Python 3.11+
 
-`pip install -e ".[reranking]"`
+## Usage
+
+The library has following components.
+
+- **AuthClient** - provides functionality to interact with data sources
```

Comment:
gRPC

Reply:
We dont use gRPC for AuthClient

Reply:
You committed at the same time I posted this message, and the line number got muddled. It's meant for the line below. You wrote gRpc instead of gRPC :)